Source/ACE.Entity/AceCharacter.cs

```diff
@@ -20,13 +20,21 @@ public AceCharacter(uint id)
 
             // Required default properties for character login
             // FIXME(ddevec): Should we have constants for (some of) these things?
-            AceObjectDescriptionFlags = (uint)(ObjectDescriptionFlag.Stuck | ObjectDescriptionFlag.Player | ObjectDescriptionFlag.Attackable);
+
+            AceObjectDescriptionFlags = (uint)(ObjectDescriptionFlag.Player | ObjectDescriptionFlag.Stuck | ObjectDescriptionFlag.Attackable);            
```

Comment:
These should probably set the field in AceCharacter, instead of call SetPropertyInt()  (I'm not sure how I didn't notice this when you first made the change).   Not sure if the change is appropriate as part of this PR, but it should be changed.